.gitlab-ci.yml

```diff
@@ -0,0 +1,34 @@
+image: 'rust:latest'
+
+stages:
+  - fmt
+  - test
+
+variables:
+  CARGO_HOME: $CI_PROJECT_DIR/cargo
+  APT_CACHE_DIR: $CI_PROJECT_DIR/apt
+  RUSTFLAGS: "-D warnings"
+
+before_script:
+  - apt-get update -yq
+  - rustup component add rustfmt-preview
```

Comment:
I believe this is changing soon to `rustfmt` no longer with the `preview`.

Reply:
Does it work now without the preview? If not, we'll have to wait to change it

Reply:
I don't think so yet, but it will happen soon, we can open an issue for it.

Reply:
Opened #1223 for it

Reply:
#1223